pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1380,4 +1380,37 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
             modalBody.insertBefore(screenReaderContent, modalBody.firstChild);
         });
     });
+
+    function trapFocus(element) {
+        var focusableElements = element.querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])'),
+        firstFocusableElement = focusableElements[0];  
+        lastFocusableElement = focusableElements[focusableElements.length - 1];
```

Comment:
Maybe you should make sure the array is not empty before indexing into it, to avoid getting `undefined`-related errors when you call `last|firstFocusableElement.focus()` down the line.

Reply:
fixed